pkg/frontend/admin_openshiftcluster_kubernetesobjects_test.go

```diff
@@ -229,124 +229,6 @@ func TestAdminKubernetesObjectsGetAndDelete(t *testing.T) {
 	}
 }
 
-func TestValidateAdminKubernetesObjectsNonCustomer(t *testing.T) {
```

Comment:
I moved this test to validate_test.go, in alignment with the function it's testing